src/callbacks_step/amr_dg1d.jl

```diff
@@ -83,30 +83,13 @@ function refine!(u_ode::AbstractVector, adaptor, mesh::TreeMesh{1},
     # actually transferring the solution to the refined cells
     refine!(u_ode, adaptor, mesh, equations, dg, cache, elements_to_refine)
 
-    # The remaining function only handles the necessary adaptation of the data structures
-    # for the parabolic part of the semidiscretization
-
-    # Get new list of leaf cells
-    leaf_cell_ids = local_leaf_cells(mesh.tree)
-
-    @unpack elements, viscous_container = cache_parabolic
-    resize!(elements, length(leaf_cell_ids))
-    init_elements!(elements, leaf_cell_ids, mesh, dg.basis)
-
     # Resize parabolic helper variables
+    @unpack viscous_container = cache_parabolic
     resize!(viscous_container, equations, dg, cache)
-
-    # re-initialize interfaces container
-    @unpack interfaces = cache_parabolic
-    resize!(interfaces, count_required_interfaces(mesh, leaf_cell_ids))
-    init_interfaces!(interfaces, elements, mesh)
-
-    # re-initialize boundaries container
-    @unpack boundaries = cache_parabolic
-    resize!(boundaries, count_required_boundaries(mesh, leaf_cell_ids))
-    init_boundaries!(boundaries, elements, mesh)
+    reinitialize_containers!(mesh, equations, dg, cache_parabolic)
```

Comment:
Use shorter re-init

Reply:
I think this was some left-over when there was no `resize!(viscous_container, equations, dg, cache)` function and one would `resize!` the parabolic helper structures in the function above, for which one would need e.g. `leaf_cell_ids = local_leaf_cells(mesh.tree)`. Maybe it was considered somewhat unnecessary to call `reinitialize_containers!(mesh, equations, dg, cache_parabolic)` at that point. 

In any case, when implementing the 1D Parabolic AMR I overlooked the possibility to shorten this by calling `reinitialize_containers!(mesh, equations, dg, cache_parabolic)`, so I am doing this in this PR(not best software engineering practice, though)